server/src/db/repositories/recipe.repository.ts

```diff
@@ -0,0 +1,47 @@
+import { ErrorUtils } from "./../../logic/utils/index";
+import { DbError, ERRORS, Recipe } from "./../../models/index";
+
+export class RecipeRepository {
+
+  private _GET_BY_ID: string = `select * 
+                                from recipes
+                                where recipes.id = $1`;
+
+  private _SAVE: string = `insert into recipes(
+                             title, userId, description,
+                             imageUrl, created)
+                           values($1, $2, $3, $4, $5)
+                           returning id`;
+
+  constructor (private _db: any) {}
```

Comment:
I have implemented this as in the example of pg-promise to get a first version running. But I also think we should introduce this type. I will change this. Maybe we should use the exact same type as the exported db value: <IDatabase<IExtensions>&IExtensions> if this will work?!